libnmstate/netapplier.py

```diff
@@ -128,6 +125,34 @@ def _apply_ifaces_state(
     validator.validate_interfaces_state(desired_state, current_state)
     validator.validate_routes(desired_state, current_state)
 
+
+def _apply_edited_ifaces_without_checkpoint(
```

Comment:
This method does not need the arguments for checkpoint handling like `commit` and `rollback_timeout`

Reply:
Also, does not need verify_change in the new approach. Done, thanks!